src/tests/topp/SpecLibSearcher_1.idXML

```diff
@@ -10,8 +10,14 @@
 		</ProteinIdentification>
 		<PeptideIdentification score_type="ZhangSimilarityScore" higher_score_better="true" significance_threshold="0" >
 			<PeptideHit score="0.999999997016446" sequence="AADDKEAC(Carbamidomethyl)FAVEGPK" charge="2" protein_refs="PH_0" >
-				<UserParam type="float" name="RT" value="-1"/>
-				<UserParam type="float" name="MZ" value="805.37978"/>
+				<UserParam type="float" name="lib:RT" value="-1"/>
+				<UserParam type="float" name="lib:MZ" value="805.379776466771"/>
+				<UserParam type="int" name="isotope_error" value="0"/>
+			</PeptideHit>
+			<PeptideHit score="0.999999997016446" sequence="AADDKEAC(Carbamidomethyl)FAVEGPK" charge="2" protein_refs="PH_0" >
+				<UserParam type="float" name="lib:RT" value="-1"/>
+				<UserParam type="float" name="lib:MZ" value="805.379776466771"/>
+				<UserParam type="int" name="isotope_error" value="1"/>
```

Comment:
of course the question is how to report this, maybe we should try and only report each hit once even if it was found with multiple isotope error queries?

Reply:
This assumes an isotopic misassignment on the experimental spectra (not the library spectra). Database search engines do it that way and report multiple PSMs for the single spectrum so I guess it should be fine for lib searches too